bb.edn

```diff
@@ -1,5 +1,38 @@
 {:tasks
- {test:clj (apply clojure "-M:test" *command-line-args*)
+ {test:clj {:doc "Run jvm tests, optionally specify clj-version (ex. :clj-1.10 :clj-1.11(default) or :clj-all)"
+            :requires ([clojure.string :as str]
+                       [clojure.edn :as edn])
+            :task (let [args *command-line-args*
```

Comment:
My rule of thumb is to not write more than 5 lines of code in a task file, or I'll move it to a file. E.g. `$project_root/bb/my_tasks.clj`

Reply:
Me too, typically! Can do!